### ISSUE TITLE
`fasterq-dump` creates various metadata files that remain in directory after completion

### ISSUE THREAD

**Issue by Alfredo-Enrique** [ncbi/sra-tools#787]
Hello SRA-Tools team! Hope your week is going well!

Quick question! 

Context: I used `prefetch` to get my .SRA files. Then I proceeded with  `fasterq-dump`. However during `fasterq-dump` various metadata files "NC__000001.10", _000002.11", "NC_000003.11" etc... are created in the output directory and **remain** after `fasterq-dump` is finished and created the corresponding SRR00XXX_1.fastq and SRR00XXX_2 fastq.

Question: Is this normal behavior? Or does it mean something went wrong in the `fasterq-dump` process. If no, and everything is okay,  I'm guessing these metadata files are okay to delete?

Thank-you in advance! Any input or perspective are greatly appreciated!

Best,

**Comment by wraetz** [comment]
Yes this is normal behavior. The files you are seeing are references. Some accessions (~20%) are stored in an aligned format. That means the bases that are identical to the reference are not stored in the accession itself. This is done to reduce the footprint of these accessions. A tool like fasterq-dump needs the references to produce FASTQ-output. The tool stores them in your file-system because they may be needed again ( for the next accession you are processing ). You can safely remove them...
